packages/get-safe-property-chain/package.json

```diff
@@ -14,12 +14,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://cbn-javieri@bitbucket.org/colbenson/get-safe-property-chain.git"
+    "url": "git+https://github.com/empathyco/x.git"
```

Comment:
This should be the url to the repo, without `git+ssh` or `git+https` as specify in the [documentation examples](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#repository).

Reply:
I'll fix this `repository` property in the last PR.